### DISCUSSION TITLE
Support for docVars and altChunk

### DISCUSSION THREAD

**Discussion by sarantoo** [dolanmiu/docx#1025] [Ideas / Requests]
It would be nice if **docVar** and **altChunk** are supported.

- **docVar** is handy when we need to pass some info to subsequent programs, ie. Word VBA, etc.  
- **altChunk** sometimes, we have a chunk of HTML that need to be embed in docx and let Word render it when the file is opened

**Comment by dolanmiu** [comment]
Can you provide the documentation for altChunk?

I can find it for docVars, but not for altChunk:

https://c-rex.net/projects/samples/ooxml/e1/Part4/OOXML_P4_DOCX_docVars_topic_ID0E6XWX.html

**Reply by sarantoo** [reply]
Documentation for altChunk can be found here:

https://c-rex.net/projects/samples/ooxml/e1/Part4/OOXML_P4_DOCX_altChunk_topic_ID0E5S41.html
